doc/source/dev/toolchain.rst

```diff
@@ -87,6 +88,9 @@ needs to be written using what is common in all of those 4 `NumPy releases`_.
     =================  ========================    =======================
      SciPy version      Python versions             NumPy versions
     =================  ========================    =======================
+     1.16               >=3.11, <3.14               >=1.25.2, <2.6.0
+     1.15               >=3.10, <3.13               >=1.23.5, <2.5.0
```

Comment:
`1.15.0` included `3.13.0` support from the start right? https://github.com/scipy/scipy/releases/tag/v1.15.0

Reply:
Yes, you're right. Fixed in commit 38060c4520bff73d6720c5103cba47b1a0793ff9.